tests/importance_tests/fanova_tests/test_tree.py

```diff
@@ -207,7 +206,9 @@ def test_tree_split_sizes(tree: _FanovaTree, node_index: NodeIndex, expected: Li
 def test_tree_subtree_active_features(
     tree: _FanovaTree, node_index: NodeIndex, expected: List[bool]
 ) -> None:
-    active_features: numpy.ndarray = tree._subtree_active_features[node_index] == expected
+    active_features: numpy.ndarray = numpy.array(
+        [tree._subtree_active_features[node_index] == expected], dtype=bool
+    )
```

Comment:
I think that the logic here used to be correct. It's the signature of `_precompute_subtree_active_features` that needs to be fixed.